frontend/components/Personcard.js

```diff
@@ -1,22 +1,14 @@
-import style from '../styles/Personcard.module.css'
-
 const PersonCard = ({ person }) => {
   return (
-    <article className={style.card}>
-      <div className={style.container}>
-        <img
-          src={person.photoImage.url}
-          width='240'
-          height='240'
-          preserveAspectRatio='xMidYMid slice'
-        />
-      </div>
-      <div className={style.bottom}>
-        <h5><strong>{person.FirstName} {person.LastName}</strong></h5>
-        <hr className={style.horizontalbar} />
-        <h6 className='beaverorange'>{person.personClass.Name}</h6>
+    <div className='col'>
+      <div className='card'>
+        <img src={person.photoImage.url} className='card-img-top' alt='Team member portrait' />
```

Comment:
For some reason, I am seeing Patricks Picture showing up a few pixels larger than the other students.  Please look into the CSS for this.